go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/ONSdigital/dp-api-clients-go v1.34.4
-	github.com/ONSdigital/dp-elasticsearch/v2 v2.1.1-0.20210506152743-bd1cf643799e
+	github.com/ONSdigital/dp-elasticsearch/v2 v2.1.1
```

Comment:
This is `v2.1.1` but the description of this PR says it should be `v2.1.0`. I assume the former is acceptable but thought I should check.

Reply:
Ah yes, I messed things up by replacing tags in dp-elasticsearch which is bad practice generally but also caused issues with the actual storage of the version in module mirror for go. I will change the reference to version in PR.